build/assets/custom-dictionary.txt

```diff
@@ -56,6 +56,7 @@ azithromycin
 BAL
 BALF
 Bao
+barcoding
```

Comment:
Look like you got the custom dictionary all figured out.  I'm planning to dump a separate file with all the unique misspelled words so that it's faster to add the okay words to the dictionary.